recipes/fastv/meta.yaml

```diff
@@ -1,16 +1,18 @@
-{% set version = "0.8.1" %}
-{% set sha256 = "210d098c6169393fc44c3fe13db34221cdcb2f3f5d71685fc0f88eb96ee1b406" %}
+{% set version = "0.10.0" %}
+{% set sha256 = "06acd1b57f1d3df0dbd344f7b47506b7a0b667481a36263a6ebbe3710f3d361f" %}
 
 package:
   name: fastv
   version: '{{ version }}'
 
 source:
-  url: https://github.com/OpenGene/fastv/archive/v{{ version }}.tar.gz
+  url: https://github.com/jodjo86/fastv/archive/v{{ version }}.tar.gz
```

Comment:
Have you asked the software dev to make a new release ? it seems they abandoned it ?

Reply:
Agreed, it may be good to keep the software up to date on conda, but I'd ask them to make a release first. I would avoid using a different source to the original one, it's not a good practice IMHO. Alternatively, change the recipe so that the software can be cloned and built from the original source.

Reply:
I made a request to the original author. I'll wait. If I don't have an answer, we will have to choose the less worse between using a different source or downloading sources directly. Both are discouraged by Bioconda.

Reply:
Great, now we have it https://github.com/bioconda/bioconda-recipes/pull/43922